mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -236,7 +236,7 @@ function removeTrack() {
     }
     var track = {};
     track.uri = [currentplaylist[i].uri];
-    mopidy.tracklist.remove(track);
+    mopidy.tracklist.remove({'uri':track.uri});
```

Comment:
I'm no JS wizz but doesn't this work out exactly the same? The original code works just fine on my system.